### PR TITLE
[@types/k6] Add type annotations for version v0.43.0

### PR DESCRIPTION
**Pull request by oleiade**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: **<<url here>>** not published yet
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by imiric — APPROVED** [review]
I'm not a TS user, and even less so with k6, so someone else should give the final approval, but this LGTM.

Nice work @oleiade! :clap:

**Review by ppcano — COMMENTED** [review]
minor issue

File: types/k6/test/tracing.ts [hunk 1]
```diff
@@ -0,0 +1,214 @@
+import http, {
```
Comment (ppcano) [inline types/k6/test/tracing.ts:1]: @oleiade, the test is for the experimental/tracing API, it seems the `http` import is unnecessary.

**Review by ppcano — COMMENTED** [review]
minor issue

File: types/k6/test/tracing.ts (hunk 1 above)
Comment (oleiade) [reply types/k6/test/tracing.ts:1]: Good catch, I guess in this case I've been failed by my linter 🥷 Fixing it 👍🏻

**Review by mstoykov — APPROVED** [review]

**Review by na-- — APPROVED** [review]
LGTM, though I just skimmed most of it

**Comment by mstoykov** [comment]
Ready to merge

**Pull request merged**